DataFormats/Portable/interface/PortableDeviceCollection.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DataFormats_Portable_interfacePortableDeviceCollection_h
+#define DataFormats_Portable_interfacePortableDeviceCollection_h
+
+#include <optional>
+#include <type_traits>
+
```

Comment:
Should there be
```suggestion

#ifdef DEBUG_COLLECTION_CTOR_DTOR
#include <iostream>
#endif
```
?

Reply:
Yes